spec/services/ccms/applicant_search_response_parser_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+module CCMS
+  RSpec.describe ApplicantSearchResponseParser do
+    describe 'parsing applicant search responses' do
+      let(:no_results_response_xml) { File.read("#{File.dirname(__FILE__)}/data/applicant_search_response_no_results.xml") }
+      let(:one_result_response_xml) { File.read("#{File.dirname(__FILE__)}/data/applicant_search_response_one_result.xml") }
+      let(:multiple_results_response_xml) { File.read("#{File.dirname(__FILE__)}/data/applicant_search_response_multiple_results.xml") }
+
+      it 'raises if the transaction_request_ids dont match' do
+        expect {
+          parser = described_class.new('20190301030405987654', no_results_response_xml)
```

Comment:
`parser` should be defined in a `let` and what is the significance of '20190301030405987654'?